Website/src/items/content/EdgeWebsite/home.yml

```diff
@@ -13,135 +13,161 @@ SharedFields:
 - ID: "f1a1fe9e-a60c-4ddb-a3a0-bb5b29fe732e"
   Hint: __Renderings
   Value: |
-    <r xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <r xmlns:p="p" xmlns:s="s"
```

Comment:
Please review the renderings for this page. It is not normal for the page to delete everything from its template standard values to re-add everything.

Reply:
Success! Manually updated "__Renderings".